dbms/src/Flash/tests/gtest_aggregation_executor.cpp

```diff
@@ -510,8 +557,27 @@ try
 }
 CATCH
 
-// TODO support more type of min, max, count.
-//      support more aggregation functions: sum, forst_row, group_concat
+TEST_F(ExecutorAggTestRunner, AggMerge)
+try
+{
+    std::vector<String> tables{"big_table_1", "big_table_2", "big_table_3"};
+    for (const auto & table : tables)
+    {
+        auto request = context
+                           .scan("test_db", table)
+                           .aggregation({Max(col("value"))}, {col("key")})
+                           .build(context);
+        auto expect = executeStreams(request, 1);
+        // test for one level merge and two level merge.
+        std::vector<UInt64> two_level_thresholds{0, 1, 100000, std::numeric_limits<UInt64>::max()};
```

Comment:
We only have 200 keys, are 100000 and `std::numeric_limits<UInt64>::max()` necessary?

Reply:
yes, you are right.
100000 and `std::numeric_limits<UInt64>::max()` are useless.

Reply:
updated.